libcudacxx/test/libcudacxx/std/numerics/bit/bitops.count/countr_zero.pass.cpp

```diff
@@ -45,22 +45,29 @@ __host__ __device__ constexpr bool constexpr_test()
       && cuda::std::countr_zero(cuda::std::numeric_limits<T>::max()) == 0;
 }
 
+template <typename T>
+__host__ __device__ inline void assert_countr_zero(const volatile T val, int expected)
+{
+  auto v = val;
+  assert(cuda::std::countr_zero(v) == expected);
+}
+
 template <typename T>
 __host__ __device__ void runtime_test()
```

Comment:
I believe this should be 
```suggestion
template <typename T>
__host__ __device__ TEST_CONSTEXPR_CXX14 void test(T val)
```

Reply:
It shouldn't be constexpr, but maybe this comment is out of date with the changes.